Rakefile

```diff
@@ -18,6 +18,8 @@ Hoe.spec "graphics" do
 
   dependency "rsdl", "~> 0.1"
 
+  extra_dev_deps << ['minitest-focus']
```

Comment:
You're requiring it but not declaring it as a dep: https://github.com/zenspider/graphics/blob/master/test/test_graphics.rb#L5-L6